pkg/identity/identity.go

```diff
@@ -127,7 +127,7 @@ func (pair *IPIdentityPair) IsHost() bool {
 func (pair *IPIdentityPair) PrefixString() string {
 	var suffix string
 	if !pair.IsHost() {
-		ones := net.IPv6len
+		var ones int
```

Comment:
This case looks interesting to me actually, there's two broad cases: Mask is nil, or not... then two inner cases: IPv4 or IPv6. It looks like if it's not IPv4, we could end up dereferencing a nil Mask.

Perhaps it should be something more like:

```
var ones int
if pair.Mask == nil {
    if pair.IP.To4() != nil {
        ones = net.IPv4len
    } else {
        ones = net.IPv6len
    }
} else {
    ones, _ = pair.Mask.Size()
}
```

Reply:
Yes, that looks reasonable. I also had a feeling that the if statement in the current shape is a little bit unclear.